src/pages/PageNotFound.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Navbar from "../components/Navbar";
+
+const PageNotFound = () => {
+    return(
+        <>
+            <Navbar/>
+            <br/><br/>
```

Comment:
bad practice using  tag `<br />`
use margin -top for <h1>

Reply:
Sure, this is already in progress in another PR (this was a temporarely modification)